pkg/stream/message.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2020 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stream
+
+import (
+	"bufio"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"k8s.io/klog"
+)
+
+type MessageType uint64
+
+const (
+	MessageTypeLogsConnect   MessageType = iota
+	MessageTypeExecConnect   MessageType = iota
+	MessageTypeMetricConnect MessageType = iota
+	MessageTypeData
+	MessageTypeRemoveConnect
+)
+
+func (m MessageType) String() string {
+	switch m {
+	case MessageTypeLogsConnect:
+		return "LOGS_CONNECT"
+	case MessageTypeExecConnect:
+		return "EXEC_CONNECT"
+	case MessageTypeMetricConnect:
+		return "METRIC_CONNECT"
+	case MessageTypeData:
+		return "DATA"
+	case MessageTypeRemoveConnect:
+		return "REMOVE_CONNECT"
+	}
+	return "UNKNOWN"
+}
+
+type Message struct {
+	ConnectID   uint64 // apiserver connection id
+	MessageType MessageType
+	Data        []byte // EdgedLogsConnection 或者 con 的原始数据
```

Comment:
nits: update those comments to "english"

Reply:
done